main.js

```diff
@@ -37,20 +37,15 @@ function doWork() {
     document.querySelectorAll(globalSelectors.viewCount)
   );
 
-  const articleViewDateSection = document.querySelector(globalSelectors.articleDate);
+  const articleViewDateSections = document.querySelectorAll(globalSelectors.articleDate);
 
-  if(articleViewDateSection) {
-    let rootDateViewsSection = articleViewDateSection.parentElement.parentElement.parentElement;
-
-    if(rootDateViewsSection?.children.length === 1) {
-      // we're dealing with the <time> element on a quote retweet
-      // do globalSelector query again but with 2nd result
-      rootDateViewsSection = document.querySelectorAll(globalSelectors.articleDate)[1].parentElement.parentElement.parentElement;
-    }
+  if(articleViewDateSections.length) {
```

Comment:
```suggestion
  if (articleViewDateSections.length) {
```

nit: I know that these originally had `if(`, instead of `if (`, but since this was the only section like that, I think it'd be beneficial to fix that in your PR as you've already modified these lines

(similarly, line 48 below)

Reply:
Thanks for the review @Samathingamajig! I've fixed these in the latest commit.